tests/integration/questionnaire/test_questionnaire_sign_out.py

```diff
@@ -0,0 +1,47 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestSignOut(IntegrationTestCase):
+    def test_sign_out_not_signed_in_redirects_to_signed_out_page(self):
```

Comment:
I reckon we can drop the prefix sign_out from some of these test as its class name. This one is a bit of a tongue twister and confused me initially with the 3 sign references